test/util/all_close_f.cpp

```diff
@@ -74,9 +74,11 @@ uint32_t test::float_distance(float a, float b, float min_signal)
     uint32_t a_uint_abs = (abs_value_bits_mask & a_fu.i);
     uint32_t b_uint_abs = (abs_value_bits_mask & b_fu.i);
     uint32_t min_signal_uint_abs = (abs_value_bits_mask & min_signal_fu.i);
-    if ((a_uint_abs < min_signal_uint_abs) && (b_uint_abs < min_signal_uint_abs))
+    if ((a_fu.i == 0 && b_uint_abs < min_signal_uint_abs) ||
```

Comment:
Not clear why you think this portion of the change is necessary.
Code before when `a_fu.i == 0`:
```
uint32_t a_uint_abs = (abs_value_bits_mask & a_fu.i);
// so a_uint_abs == 0 now
(a_uint_abs < min_signal_uint_abs) => true
```